packages/contractkit/src/utils/send-tx.ts

```diff
@@ -0,0 +1,50 @@
+import debugFactory from 'debug'
+import { Address } from 'src/base'
+import { toTxResult, TransactionResult } from 'src/utils/tx-result'
+import { TransactionObject } from 'web3/eth/types'
+
+const debug = debugFactory('contractkit:sendtx')
+
+export interface TxOptions {
+  gasInflationFactor?: number
+  gasBeneficiary?: Address
+  gasCurrency?: Address
+  from?: Address
+  estimatedGas?: number | undefined
+}
+
+/**
+ * sendTransaction mainly abstracts the sending of a transaction in a promise like
+ * interface.
+ */
+export async function sendTransaction<T>(
```

Comment:
What is the difference between this and `kit.sendTransaction`/`kit.sendTransactionObject`

Reply:
this should be addressed later..
one is for `raw` sendTransaction and the other to sendTx from congtract wrappers